doc/version-history.rst

```diff
@@ -10,6 +10,10 @@ Version History
 .. No new work should be required in order to complete this section.
 .. Below is an example of a version history format.
 
+v0.6.0
+======
+* Rewrite fits files to match Vera Rubin Observatory format
```

Comment:
Please make it a habit to end your commit messages in a `.` so please amend this commit.